sunspot/spec/integration/field_lists_spec.rb

```diff
@@ -31,6 +31,22 @@
     end
   end
 
+  it 'when listing existing text fields it does not raise Sunspot::UnrecognizedFieldError' do
```

Comment:
@mgadikota-rpx sorry to bother you :) but I was ment to say it should read like:
> does not raise Sunspot::UnrecognizedFieldError when listing existing text fields

Reply:
@serggl  just made this change.